x-pack/plugins/security_solution/public/detections/pages/detection_engine/chart_panels/index.test.tsx

```diff
@@ -227,79 +227,71 @@ describe('ChartPanels', () => {
 
   describe(`'Reset group by fields' context menu action`, () => {
     describe('Group by', () => {
-      const alertViewSelections = ['trend', 'table', 'treemap'];
-
-      alertViewSelections.forEach((alertViewSelection) => {
-        test(`it resets the 'Group by' field to the default value, even if the user has triggered validation errors, when 'alertViewSelection' is '${alertViewSelection}'`, async () => {
-          (useAlertsLocalStorage as jest.Mock).mockReturnValue({
-            ...defaultAlertSettings,
-            alertViewSelection,
-          });
+      test(`it resets the 'Group by' field to the default value, even if the user has triggered validation errors, when 'alertViewSelection' is 'treemap'`, async () => {
+        (useAlertsLocalStorage as jest.Mock).mockReturnValue({
+          ...defaultAlertSettings,
+          alertViewSelection: 'treemap',
+        });
```

Comment:
It seems we no longer test the `trend` and `table` sections, is this correct?

Reply:
As the count table and the trend chart is now rendered by Embeddable, the original test cases are not longer fit.  I have updated their tests accordingly.